src/apps/user/schemas.py

```diff
@@ -9,11 +9,11 @@ class UserLoginInputSchema(BaseModel):
 
 
 class UserBaseSchema(BaseModel):
-    first_name : str = Field(max_length=50)
-    last_name : str = Field(max_length=75)
-    email : str = Field(max_length=50)
-    birth_date : datetime.date
-    username : str = Field(max_length=50)
+    first_name : Optional[str] = Field(max_length=50)
```

Comment:
Ta zmiana  rozpierniczy ci rejestracje, bo ci na wszyskie kolumny moga nulle powlatywac,
raczej powinny byc to osobne schemy + imo zmiana maila nie powinna byc taka prosta, albo brak mozliwosci zmiany imo albo confirmation, chociaz z drugiej strony na rejestracji tez confirmation nie ma hmm, up to you jak ma byc

Reply:
no tutaj za daleko poszedłem, to fakt XD

Reply:
ale confirmation to już będzie inny PR, dobry pomysł btw